### DISCUSSION TITLE
Add ko-fi widget

### DISCUSSION THREAD

**Discussion by AegisBlue** [nunocoracao/blowfish#604] [Ideas]
Hello!

I made a [discussion](https://github.com/nunocoracao/blowfish/discussions/594) asking for the location of buymeacoffee widget so I can implement a ko-fi version and the answer is perfect!

Although the implementation worked, I took it a step further and made it configurable in params.toml.

I was wondering if I can create a pull request with the feature as per the rule of CONTRIBUTING.md.

This will be my first public pull request ever on github so I might make some mistakes haha.

**Comment by nunocoracao** [comment]
@AegisBlue don't worry go for it
